.github/ISSUE_TEMPLATE/feature_request.md

```diff
@@ -0,0 +1,17 @@
+---
+name: Feature request
```

Comment:
nit: request -> Request

Reply:
Since it's heading, we use same first letter capitalisation below in `SDK Issue`